setup/system.sh

```diff
@@ -36,7 +35,7 @@ hostname $PRIMARY_HOSTNAME
 # See https://www.digitalocean.com/community/tutorials/how-to-add-swap-on-ubuntu-14-04
 # for reference
 
-SWAP_MOUNTED=$(cat /proc/swaps | tail -n+2)
+SWAP_MOUNTED=$(< /proc/swaps tail -n+2)
```

Comment:
This works - but for readability, I suggest:
```suggestion
SWAP_MOUNTED=$(tail -n+2 /proc/swaps)
```

Reply:
OK, good change!